assets/app.js

```diff
@@ -607,11 +536,20 @@ new Vue({
             }
         },
         translationsBaseProducts: function(){
+            if(this.rawBaseProduct == null)
+            this.rawBaseProduct = BaseProductOptions;
```

Comment:
indentation trouble? that's not indented properly. also make sure you use `{` and `}` even if it's not required, we've faced when working with this type of statement, we always end up making some minor mistakes.